crypto-ffi/bindings/js/src/CoreCryptoInstance.ts

```diff
@@ -272,8 +148,15 @@ export class CoreCrypto {
      * **CAUTION**: This {@link CoreCrypto} instance won't be usable after a call to this method,
      * but there's no way to express this requirement in TypeScript, so you'll get errors instead!
      */
-    async close() {
-        await CoreCryptoError.asyncMapErr(this.#cc.close());
+    close() {
+        if (CoreCryptoFfi.instanceOf(this.#cc)) {
+            this.#cc.close();
+            this.#cc.uniffiDestroy();
+        } else {
+            throw new Error(
+                "CoreCrypto hasn't been initialized properly. Context: closing"
```

Comment:
Suggestion: here and in E2ei, we change the error text: "Use after Free: CoreCrypto`, which is simultaneously shorter and more descriptive.